client/components/plans/plan-header/index.jsx

```diff
@@ -3,11 +3,19 @@
  */
 import classNames from 'classnames';
 import React from 'react';
+import noop from 'lodash/noop';
 
 const PlanHeader = React.createClass( {
 	propTypes: {
 		isPlaceholder: React.PropTypes.bool,
-		text: React.PropTypes.string
+		text: React.PropTypes.string,
+		onClick: React.PropTypes.func
+	},
+
+	getDefaultProps() {
+		return {
+			onClick: noop
```

Comment:
I am in the camp of people who would rather just do `() => {}` and avoid importing a dependency for noops, but it's not a big deal.